updates/27/updates.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE common_asset_details SET cryptocompare="DAI" WHERE identifier="eip155:1/erc20:0xab7FA2B2985BCcfC13c6D86b1D5A17486ab1e04C";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xab7FA2B2985BCcfC13c6D86b1D5A17486ab1e04C", "FARM_DAI", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xab7FA2B2985BCcfC13c6D86b1D5A17486ab1e04C", "A", 1, "0xab7FA2B2985BCcfC13c6D86b1D5A17486ab1e04C", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0xab7FA2B2985BCcfC13c6D86b1D5A17486ab1e04C", "fDAI", "dai", "DAI", NULL, 1603116059, NULL);
+UPDATE common_asset_details SET cryptocompare="DAI" WHERE identifier="eip155:43114/erc20:0xd586E7F844cEa2F87f50152665BCbc2C279D8d70";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:43114/erc20:0xd586E7F844cEa2F87f50152665BCbc2C279D8d70", "Dai Stablecoin", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:43114/erc20:0xd586E7F844cEa2F87f50152665BCbc2C279D8d70", "A", 43114, "0xd586E7F844cEa2F87f50152665BCbc2C279D8d70", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:43114/erc20:0xd586E7F844cEa2F87f50152665BCbc2C279D8d70", "DAI.e", "dai", "DAI", NULL, 1627052370, NULL);
+UPDATE common_asset_details SET cryptocompare="DAI" WHERE identifier="eip155:56/erc20:0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3", "Dai Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3", "A", 56, "0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3", "DAI", "dai", "DAI", NULL, 1599646888, NULL);
+UPDATE common_asset_details SET cryptocompare="USDT" WHERE identifier="eip155:56/erc20:0x55d398326f99059fF775485246999027B3197955";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0x55d398326f99059fF775485246999027B3197955", "Tether USD", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0x55d398326f99059fF775485246999027B3197955", "A", 56, "0x55d398326f99059fF775485246999027B3197955", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0x55d398326f99059fF775485246999027B3197955", "USDT", "tether", "USDT", NULL, 1599201028, NULL);
```

Comment:
this one doesn't seem to be USDT

![image](https://github.com/user-attachments/assets/11f070c3-7354-4291-a89a-519fe79f7c80)